web/packages/api/src/status.ts

```diff
@@ -72,8 +72,8 @@ export const BlockLatencyThreshold = {
 }
 
 export const InsufficientBalanceThreshold = {
-    // Minimum as 10 DOT
-    Substrate: 100_000_000_000,
+    // Minimum as 100 DOT
+    Substrate: 1_000_000_000_000,
     // Minimum as 0.1 Ether
     Ethereum: 100_000_000_000_000_000,
```

Comment:
I changed all the tenderly alerts to alert if lower than 0.3 ETH. https://linear.app/snowfork/issue/SNO-1067#comment-d4b0c3c5

Maybe we want to do the same here.

Reply:
https://github.com/Snowfork/snowbridge/pull/1249/commits/5c0137ed53f9ccc67d73730c0de626a1d3f5d6d2